src/mixins/java/org/spongepowered/common/mixin/api/mcp/server/level/ServerPlayerMixin_API.java

```diff
@@ -117,6 +119,8 @@
     @Shadow public abstract net.minecraft.server.level.ServerLevel shadow$getLevel();
     // @formatter:on
 
+    private PermissionChecker adventure$permissions;
```

Comment:
This should be `api$`, not `adventure$` - all fields in an API mixin should be prefixed that way, not with the library it comes from.